diff --git a/tests/cefclient/browser/osr_window_win.h b/tests/cefclient/browser/osr_window_win.h
index 3b2ed53..1775665 100644
--- a/tests/cefclient/browser/osr_window_win.h	
+++ b/tests/cefclient/browser/osr_window_win.h
@@ -169,7 +169,7 @@ class OsrWindowWin
 
   const OsrRendererSettings settings_;
   HWND hwnd_;
-  std::unique_ptr<OsrRenderHandlerWin> render_handler_;
+  scoped_ptr<OsrRenderHandlerWin> render_handler_;
 
   // Class that encapsulates IMM32 APIs and controls IMEs attached to a window.
   scoped_ptr<OsrImeHandlerWin> ime_handler_;
